docs/reference/getting-started.asciidoc

```diff
@@ -352,7 +352,7 @@ And the response:
 health status index    uuid                   pri rep docs.count docs.deleted store.size pri.store.size
 yellow open   customer 95SQ4TSUT7mWBT7VNHH67A   5   1          0            0       260b           260b
 --------------------------------------------------
-// TESTRESPONSE[s/95SQ4TSUT7mWBT7VNHH67A/.+/ s/260b/\\d+b/ _cat]
+// TESTRESPONSE[s/95SQ4TSUT7mWBT7VNHH67A/.+/ s/260b/\\d+(\\.\\d+)?b/ _cat]
```

Comment:
shouldn't we also cover the case where the unit changes from `b` to `kb` ?

Reply:
does `\d+(\.\d)?[kbg]?b` sound better? With that `1.1b`, `100b`, `12.1gb`, `1.1kb` should match

Reply:
I think restricting it to `kb` and `b` only for this test would be better, we should detect if the value suddenly jumps to `mb` here for an almost empty index. So adding `k?` to the mix should be enough IMHO.

Reply:
@cbuescher ++ that's what I had in mind too.